release.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+import argparse
+import os
+import subprocess
+
+from build_and_push_images import IMAGES, docker_build, docker_push
+
+
+def ask_yes_no(question):
+    if input(question).lower().strip()[0] == "y":
+        return True
+    return False
```

Comment:
There's an ~~app~~[standard library function](https://docs.python.org/3/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool) for this.